documentation/examples/cqrs/web-client/src/main/scala/cqrs/webclient/Main.scala

```diff
@@ -51,7 +48,7 @@ object Main {
       }
     }
 
-    def onAddValueClicked(meter: Meter): Event => Unit = _ => {
+    def onAddValueClicked: Observer[Meter] = Observer { meter =>
```

Comment:
This can be just `def onAddValueClicked(meter: Meter): Unit = {` (usage would be exactly the same), or if you want to keep it as Observer it can be a `val` instead of `def`